### PR TITLE
Update project endpoint with profile

### PR DESCRIPTION
**Pull request by hanifvirani**
* Add missing url field.
* Add missing groups and users fields.
* Add new profile field.

### CODE REVIEW

File: source/includes/projects.md [hunk 1]
```diff
@@ -115,22 +148,39 @@ Authorization: Token "YOUR SDE ACCESS TOKEN"
...
+        "role": "PR4"
+    }]
 }
 ```
 
 Fields      | Required | Description
 ------------|----------|-------------
-application | Yes      | The ID of the application the project belongs to
-name        | Yes      | The name of the new project
+application | Yes      | The ID of the application the project should be created under.
```
Comment (hanifvirani) [inline source/includes/projects.md]: Not sure if I should group required and non required together or leave this alphabetically ordered.

File: source/includes/projects.md [hunk 2]
```diff
@@ -115,22 +148,39 @@ Authorization: Token "YOUR SDE ACCESS TOKEN"
...
+groups      | No       | A list of dictionaries per group that are to be assigned to the project. Each dictionary should contain the group's id and the desired role.
```
Comment (hanifvirani) [inline source/includes/projects.md]: This is currently one long line and github is formatting it weird. There might be a better way to format it.

File: source/includes/projects.md (hunk 1 above)
Comment (funkaoshi) [inline source/includes/projects.md]: I'd stick with alphabetical for now. We might want to bold required fields or something?

**Comment by funkaoshi** [comment]
Do we normally call things 'dictionaries'? I guess that's what they are.

I think this is fine. So people can query the profile endpoint to pick a profile, and then set one on the project by passing it in? And if they change it via the API it will reset the profile, the same way we do in the application? Might want co comment on that behaviour.

**Comment by hanifvirani** [comment]
I noticed that we are missing the Update a Project and Delete a Project part of the docs. We can mention the updating profile behavior there.

**Comment by hanifvirani** [comment]
> Do we normally call things 'dictionaries'? I guess that's what they are.

Alternative would be something like - "A list of user objects each consisting of.."

**Pull request merged**